packages/app/src/components/Header/index.tsx

```diff
@@ -25,39 +28,77 @@ const Header = ({ data }: { data: any }) => {
   const importError = useRecoilValue(importErrorAtom);
   
   const { provenance, isAtRoot, isAtLatest } = useContext(ProvenanceContext);
-
+  
   const [ attributeDialog, setAttributeDialog ] = useState(false);
   const [ showImportModal, setShowImportModal ] = useState(false);
-  const [ showDataTable, setShowDataTable ] = useState(false);
   const [ isMenuOpen, setIsMenuOpen ] = useState(false);
+  const [ loginMenuOpen, setLoginMenuOpen ] = useState(false);
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>();
+  
+  const visibleSets = provenance.getState().visibleSets;
+  const hiddenSets = provenance.getState().allSets.filter((set: string) => !visibleSets.includes(set));
 
   const handleImportModalClose = () => {
     setShowImportModal(false);
   }
 
   const handleMenuClick = (event: React.MouseEvent<any>) => {
-    setAnchorEl(event.currentTarget);
+    handleMenuOpen(event.currentTarget);
+  };
```

Comment:
This is another candidate for clean up where we can use `(event) => handleMenuOpen(event.currentTarget)` in the JSX